Adds information related to the license and the cluster limitation (#4044)
mattermost/docs: source/install/mattermost-omnibus.rst

Mattermost Omnibus is a Debian package that packages the individual components of a Mattermost deployment into a single installation process. The package leverages the ``apt`` package manager to install and keep the components of the platform updated, using a custom CLI and ansible recipes to link those components together and configure them.

> Mattermost Omnibus currently supports Ubuntu's ``bionic`` and ``focal`` distributions. Support for RedHat/CentOS distributions will be available in future versions. The package bundles Mattermost Enterprise Edition.

Reviewer: ```suggestion
Mattermost Omnibus currently supports Ubuntu's ``bionic`` and ``focal`` distributions. Support for RedHat/CentOS distributions will be available in future versions. The package bundles the free, unlicensed Mattermost Enterprise Edition.
```

> Mattermost Omnibus currently supports Ubuntu's ``bionic`` and ``focal`` distributions. Support for RedHat/CentOS distributions will be available in future versions. The package bundles the E0 Mattermost platform version.

Reviewer: I don't think we refer to E0 publicly. @wiersgallak, @jasonblais, can you confirm?

A replacement could be "The package bundles the Mattermost Enterprise Edition platform version."

Reviewer 2: Yes, "Mattermost Enterprise Edition" is what we refer to it by, but I wonder if there's a way we can word it such that it's clear it's the free version (cc @wiersgallak)

Reviewer: Thanks, @jasonblais. I reworded the sentence but was considering whether "The package bundles the unlicensed Mattermost Enterprise Edition platform version." would work.

Reviewer 2: How about `The package bundles the free, unlicensed Mattermost Enterprise Edition.`

> Mattermost Omnibus currently supports Ubuntu's ``bionic`` and ``focal`` distributions. Support for RedHat/CentOS distributions will be available in future versions. The package bundles the E0 Mattermost platform version.

Reviewer: ```suggestion
Mattermost Omnibus currently supports Ubuntu's ``bionic`` and ``focal`` distributions. Support for RedHat/CentOS distributions will be available in future versions. The package bundles Mattermost Enterprise Edition.
```

With Mattermost Omnibus, the ``config.json`` file is no longer used as `Omnibus stores the Mattermost configuration in the database <https://docs.mattermost.com/administration/config-in-database.html>`__. The Omnibus platform itself requires of a configuration of its own, that is stored in ``/etc/mattermost/mmomni.yml``. This file contains the data that Omnibus needs to configure the platform and connect all the services together. So you’ll need to use mmctl to make changes to your Mattermost server configuration using ``mmctl --local config edit``.

> For Omnibus to work properly, there are some configuration parameters that are fixed and cannot be changed through the web interface - for example, the port that Mattermost uses to run. Other parameters need to be configured directly in the ``mmomni.yml`` file instead of in the Mattermost web interface or the configuration.

Reviewer: ```suggestion
For Omnibus to work properly, there are some configuration parameters that are fixed and cannot be changed through the web interface - for example, the port that Mattermost uses to run. Other parameters need to be configured directly in the ``mmomni.yml`` file instead of in the Mattermost web interface or the ``config.json`` file.
```

- ``mmomni tail``: Runs a join tail of logs of all Omnibus components.

Can I install without a domain?
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

Although the recommended way to install and configure Omnibus is with SSL enabled, if you want to use or test without it, you can run:

> ``sudo MMO_HTTPS=false apt install mattermost-omnibus``.

Reviewer: ```suggestion
``sudo MMO_HTTPS=false apt install mattermost-omnibus``
```

Yes! We are planning several packages and currently seeking feedback to help us prioritize these.

Are there plans to support other OS distros?
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

Yes! We are currently seeking feedback to help us prioritize these.

Can I use MySQL instead of PostgreSQL?
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

MySQL is not supported. Omnibus is architected to run with PostgreSQL.

Can I use a license with Omnibus?
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> Yes! Mattermost Omnibus bundles the E0 Mattermost version, so the enterprise features can be unlocked uploading a license.

Reviewer: ```suggestion
Yes. Mattermost Omnibus bundles Mattermost Enterprise Edition, and Enterprise features are unlocked when you purchase and upload a license.
```

Can I use an Omnibus server as part of a cluster?
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> No, Omnibus has been designed to be a self contained Mattermost platform, so it contains and expects all the necessary components to be in the same server.

Reviewer: ```suggestion
No, Omnibus is designed to be a self-contained Mattermost platform, so it expects all the necessary components to be in the same server.
```
